tinyms/serving/client/client.py

```diff
@@ -69,4 +70,13 @@ def predict(img_path, servable_name, dataset_name="mnist"):
     elif res_body['status'] != 0:
```

Comment:
Please add the strategy as the parameter of `predict` method and the default one is `TOP1_CLASS`.

Reply:
done, strategy is added in predict, and the output format and score are both rendered